pom.xml

```diff
@@ -139,17 +137,6 @@
             </exclusions>
         </dependency>
 
-        <dependency>
-            <groupId>xml-apis</groupId>
-            <artifactId>xml-apis</artifactId>
-            <version>${xml-apis.version}</version>
-        </dependency>
```

Comment:
I verified that with the new build toolchain `stax-api` is correctly bundled with the plugin (it wasn't before) so this is no longer needed.